test/pipelines/test_standard_pipelines.py

```diff
@@ -213,6 +246,37 @@ def test_most_similar_documents_pipeline_batch(retriever, document_store):
     document_store.write_documents(documents)
     document_store.update_embeddings(retriever)
 
+    docs_id: list = ["a", "b"]
+    filters = {"source": ["wiki3", "wiki4", "wiki5"]}
+    pipeline = MostSimilarDocumentsPipeline(document_store=document_store)
+    list_of_documents = pipeline.run_batch(document_ids=docs_id, filters=filters)
```

Comment:
Could you move this `run_batch` call with the `filters` option specified to under the test you made called `test_most_similar_documents_pipeline_with_filters_batch`? I think you misplaced it in the normal batch test.